matrix-json.c

```diff
@@ -128,3 +130,121 @@ const gchar *matrix_json_array_get_string_element(JsonArray *array,
 	element = matrix_json_array_get_element(array, index);
 	return matrix_json_node_get_string(element);
 }
+
+/* Arrays of name/node for an object */
+struct CanonicalObjectTmp {
+  guint total, count;
```

Comment:
Done, I've added a patch to remove tabs in matrix-json and made all this 4 space